src/command.ts

```diff
@@ -313,6 +313,7 @@ export class Command {
     }
 
     if (getInheritedOption(options, "json")) {
+      options.interactive = false;
```

Comment:
This is a real logic bug but it seems to only affect one command. And that command usually isn't called with `--json`.

https://github.com/firebase/firebase-tools/blob/a952fb4067891bfd18c1c0fccecd82dc5dd7a8d5/src/init/features/aitools/promptUpdater.ts#L89